src/sentry/eventstore/base.py

```diff
@@ -172,6 +176,22 @@ def get_events(
         """
         raise NotImplementedError
 
+    def get_events_snql(
+        self,
+        organization_id: int,
+        group_id: int,
+        start: Optional[datetime],
+        end: Optional[datetime],
+        conditions: Sequence[Condition],
+        orderby: Optional[Sequence[str]] = None,
```

Comment:
Should we make this required? Or do we actually need a parameter here at all?

Reply:
Yeah, I think that makes sense. The initial thought was to keep it generic and copy the method signature over from the other `get_events` method but I'm not sure its useful in this case.